src/app/components/Text/index.test.jsx

```diff
@@ -20,9 +20,9 @@ describe('Text', () => {
 
     const data = {
       blocks: [
-        paragraphBlock('01', 'This is a paragraph.'),
-        paragraphBlock('02', 'This is another paragraph.'),
-        paragraphBlock('03', 'This is a final paragraph.'),
+        paragraphBlock('01', 'It was designed by London-based florist Philippa Craddock, who also created the floral displays for St George\'s Chapel and St George\'s Hall using locally sourced foliage, [which were later donated to local hospices](/news/articles/c000000000ro).'),
```

Comment:
I think we need to make sure we cover all possible markdown markup (hope that makes sense) in these tests as they are so core to our rendering. (This is a comment to ensure others check, not a request for changes)

Reply:
@benjaminhobbs and @bbc/article-devs-news I've added more examples of md, but unsure on how much more to add as we will not process full markdown. Thoughts?

Reply:
@jtart It would be good to have tests that capture combinations - for example, italic and bold together, within a link. 
Separately, we'll need to consider how we'll handle the styling of these -- if a journalist uses these together, will we even want to display a link that has bold and italic typography? Probably not. We should work with UX to discuss these different cases.